extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/text_translation.py

```diff
@@ -129,12 +129,19 @@ def construct_stream_output(
             "metadata": {},
         }
     )
+
     accumulated_message = ""
     for new_text in streamer:
         if isinstance(new_text, str):
+            # For some reason these symbols aren't filtered out by the streamer
+            new_text = new_text.replace("</s>", "")
+            new_text = new_text.replace("<s>", "")
+            new_text = new_text.replace("<pad>", "")
```

Comment:
Is this needed in the `summarization` model parser too? That one only replaces `</s>` and `<s>`

Reply:
I didn't personally see it in `summarization`. If it does show up, we can fix forward but this was just me doing some manual testing so could be I missed some use cases